apisix/cli/ops.lua

```diff
@@ -407,7 +413,12 @@ Please modify "admin_key" in conf/config.yaml .
 
             if ip_port_to_check[addr] == nil then
                 table_insert(listen_table,
-                        {ip = ip, port = port, enable_http2 = enable_http2})
+                        {
```

Comment:
can you explain the usage of this block briefly? And why this change is needed.

Reply:
because I want to add a new field `enable_quic`. If I add `enable_quic` directly, the length of this line will too long to cause a lint error.